runtime/parachains/src/runtime_api_impl/v1.rs

```diff
@@ -332,6 +332,6 @@ pub fn validation_code_by_hash<T: paras::Config>(
 }
 
 /// Disputes imported via means of on-chain imports.
-pub fn on_chain_votes<T: paras_inherent::Config>() -> Option<ScrapedOnChainVotes<T::Hash>> {
+pub fn on_chain_votes<T: paras_inherent::pallet::Config>() -> Option<ScrapedOnChainVotes<T::Hash>> {
```

Comment:
this diff shouldn't be neccesary since we re-export contents of pallet

Reply:
true, residue of a duplicate `Config` re-export where `rustc` failed to resolve the right one accoring to `trait` bounds. Rust issue pending.